README.adoc

```diff
@@ -35,6 +36,33 @@ NOTE: If you want to use reveal.js 2, please switch your asciidoctor-reveal.js r
 
 TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by keeping your source files on `master` branch and all output files on the `gh-pages` branch.
 
+[[node-package]]
+== Node package
+
+Using npm:
+
+ $ npm i -g npm
+ $ npm i --save asciidoctor.js
+ $ npm i --save asciidoctor-reveal.js
```

Comment:
```
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/asciidoctor-reveal.js
```

So how can I test locally?

I tried using `npm link`, even as root (which I hate doing) and it always failed because of a missing dependency on `asciidoctor-template.js`:

```
npm ERR! 404  'asciidoctor-template.js' is not in the npm registry.
```

So I tried doing a `sudo npm link` inside a checkout of asciidoctor-template.js and it worked but a subsequent `sudo npm link` from asciidoctor-reveal.js still failed like if asciidoctor-template.js was not installed:

```
npm ERR! 404  'asciidoctor-template.js' is not in the npm registry.
```

Sorry for my lack of experience but I tried. Can you help me here?